app/create_db.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 import re
+import unicodedata
```

Comment:
Does this need added to requirements?

Reply:
No, it's a standard Python library. Nothing additional needs to be installed.

Reply:
sounds good, just wasn't sure